README.md

```diff
@@ -38,6 +38,12 @@ Backend | RESTful API backend for Casnode | Golang + Beego + MySQL | https://git
 
 
 - Setup database:
+    Create application config file from sample

```

Comment:
This configuration-related part seems more appropriate to write in `Custom settings` or `Run backend` part?

Reply:
Configuration comes from the same place, which can simplify project release and deployment